pages/assessment.tsx

```diff
@@ -1,55 +1,58 @@
 import type { NextPage } from 'next'
-import Pointsbox from '../components/pointsbox'
 import styles from '../styles/assessment.module.css'
+import Textbox from "../components/textbox";
+import { useState } from 'react';
+import data from '../data/IT2810Høst2018.json'
+import cleanHtmlText from "../functions/helpFunctions";
+
 
 const Assessment: NextPage = () => {
+
+  //fw: create new logic to get data
+  const answer_0: string = cleanHtmlText(data.ext_inspera_candidates[0].result.ext_inspera_questions[0].ext_inspera_candidateResponses[0].ext_inspera_response!)
+  const answer_1: string = cleanHtmlText(data.ext_inspera_candidates[1].result.ext_inspera_questions[0].ext_inspera_candidateResponses[0].ext_inspera_response!)
+  const answer_2: string = cleanHtmlText(data.ext_inspera_candidates[2].result.ext_inspera_questions[0].ext_inspera_candidateResponses[0].ext_inspera_response!)
+  const answer_3: string = cleanHtmlText(data.ext_inspera_candidates[3].result.ext_inspera_questions[0].ext_inspera_candidateResponses[0].ext_inspera_response!)
+  const answer_4: string = cleanHtmlText(data.ext_inspera_candidates[4].result.ext_inspera_questions[0].ext_inspera_candidateResponses[0].ext_inspera_response!)
+  const answer_5: string = cleanHtmlText(data.ext_inspera_candidates[5].result.ext_inspera_questions[0].ext_inspera_candidateResponses[0].ext_inspera_response!)
+  const answer_6: string = cleanHtmlText(data.ext_inspera_candidates[6].result.ext_inspera_questions[0].ext_inspera_candidateResponses[0].ext_inspera_response!)
+  
+
+  const answers: string[] = [answer_0, answer_1, answer_2, answer_3, answer_4, answer_5, answer_6]
+  
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [maxItemsPerPage, setMaxItemsPerPage] = useState<number>(4); //max items set to 4 as default
+
+
+  const changePage = (direction: string) : void => {
+    if (direction == 'back') {
+      setCurrentPage(currentPage - 1) 
+    } else if (direction == 'next') {
+      setCurrentPage(currentPage + 1)
+    }
+  }
+
   return (
     <div className={styles.container}>
         <h1>
-            Eksamen TDT100
+            {data.ext_inspera_assessmentRunTitle}
         </h1>
       <main className={styles.main}>
-        <div className={styles.wrapper}>
-        <div className={styles.grid}>
-          <div className={styles.card}>
-            <div className={styles.alignTitlePoints}>
-              <h2>Card 1 </h2>
-              <Pointsbox maxPoints={4}/>
-            </div>
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+          <div className={styles.grid}>
+            {/* slice(start, end) */}
```

Comment:
Kanskje litt lite beskrivende kommentar?:)

Reply:
Hahah ups, skal fjernes. Hadde den for å huske hvordan slice funket :')